### PR TITLE
xeno figurines

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
add 2 xeno figurines:
- xippy
- warrior (xx-616)

and a xeno figurine spawner that can be mapped, until then it can be spawned by admins :trollface:

## Why / Balance
honour the brave xenos that got us where we are today :saluting_face: 

## Media
xippy and 616 with friends

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
no xl no fun

### CODE REVIEW

**Comment by Kadeo64** [comment]
We should also add the other warriors that have slain the entire almayer before (prosmasher)

**Comment by deltanedas** [comment]
dont worry it was a prosmasher alt account

File: Resources/Prototypes/_RMC14/Entities/Objects/xeno_figurines.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# these dont get spawned by patron figurine spawner since no PatronFigurineComponent
+# map in "xeno figurine spawner" or spawn them midround :)
+
+- type: entity
+  abstract: true
+  parent: RMCBaseFigurinePatron
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Objects/xeno_figurines.yml:6]: I got spooked by this but apparently this is fine
Should probably rename it to not be FigurinePatron some day

**Review by DrSmugleaf — APPROVED** [review]

File: Resources/Prototypes/_RMC14/Entities/Objects/xeno_figurines.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/_RMC14/Entities/Objects/xeno_figurines.yml:6]: yes the base figure actually has no patron specific stuff

**Pull request merged**